Platform.PCL/Realm.PCL/RealmPCL.cs

```diff
@@ -293,13 +293,15 @@ public void Write(Action action)
         }
 
         /// <summary>
-        /// Execute an action inside a temporary <see cref="Transaction"/> on a worker thread. If no exception is thrown,
+        /// Execute an action inside a temporary <see cref="Transaction"/> on a worker thread, <b>if</b> called from UI thread. If no exception is thrown,
```

Comment:
Maybe "Execute an action in a temporary transaction. If called from the UI thread, this will take place on a worker thread."